punnetsquaremaker.py

```diff
@@ -39,40 +60,77 @@ def make_table(parent1, parent2):
 	return table
 
 def print_table(table, c1, c2): # formats and prints Punnett square
+	f= open(name, 'a') # open the file
 	divlength = (len(c1[0])*2+4)*2**(len(c1[0]))
+	print ''
 	print '',
 	for a in c2:
 		print ' '*(len(c1[0])+3) + a + '',
+		f.write('& ' + a + ' ')
 	print '\n' + ' '*(len(c1[0])+1) + '-'*(divlength)
-	for row in table:
+	f.write('\\\ \n\\hline\n')
+	
+	for i, row in enumerate(table):
 		print c1[table.index(row)],
+		f.write(c1[table.index(row)] + ' & ')
 		print '|',
-		for cell in row:
+		for j, cell in enumerate(row):
 			print cell + ' | ',
+			if j != len(row)-1:
+				f.write(cell + ' & ')
+			else:
+				f.write(cell + ' ')
 		print '\n' + ' '*(len(c1[0])+1) + '-'*(divlength)
-
+		if i != len(table)-1:
+			f.write('\\\ \n')	
+	
 def print_genotype_frequencies(table): # calculates frequencies for each genotype present in table
+	f= open(name, 'a') # open the file
+	f.write('\n')
 	calculated = []
 	genotypes = [a for b in table for a in b]
-	for x in genotypes:
+	for k, x in enumerate(genotypes):
 		count = 0
 		for y in genotypes:
 			if sorted(x) == sorted(y):
 				count += 1
 		if sorted(x) not in calculated:
 			print "The frequency of the " + x + " genotype is " + str(float(count)/float((len(genotypes)))*100) + "%."
+			f.write(x + ' & ' + str(float(count)/float((len(genotypes)))*100) + '\\% \\\ \\hline \n')				
 		calculated.append(sorted(x))
 
-
-print 'Hello, and welcome to the Punnett square maker! To get started, enter the genotype of the first parent. There should be two alleles for each gene, and each should be represented by one letter. The genes should be separated by spaces. For example, a valid genotype would be "Xx Yy zz", while "XxYyZz" would not.'
+print '' 
+print '==========   Punnett square maker & Latex table export  =============='
+print '' 
+print 'Hello, and welcome to the Punnett square maker! To get started, enter the genotypes of each parent. There should be two alleles for each gene, and each should be represented by one letter.'
+print 'The genes should be separated by spaces. For example, a valid genotype would be "Xx Yy zz", while "XxYyZz" or "Xx Yy zz " would not.'
+print '' 
+print '====================================================================='
+print ''
 while True:
+	now = datetime.datetime.now() # Date & time
+	name = '%s_table.tex'%(now.strftime("%Y-%m-%d_%H-%M-%S")) # Give a name for the file 
 	p1 = raw_input("Please enter the genotype of the first parent: ").split(' ')
 	p2 = raw_input("Please enter the gentype of the second parent: ").split(' ')
+	headtex()
 	c1 = get_all_combinations(p1)
 	c2 = get_all_combinations(p2)
+	
+	f= open(name, 'a')
```

Comment:
Please consider making the writing of a LaTex file optional--i.e., giving the user the ability to use the tool from the command line without creating a new file.